src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -389,8 +389,8 @@ def _deploy_arm_template_core_unmodified(cmd, resource_group_name, template_file
 
 
 class JsonCTemplate:
-    def __init__(self, template_as_bytes):
-        self.template_as_bytes = template_as_bytes
+    def __init__(self, template_data):
+        self.template_data = template_data
```

Comment:
The `template_as_bytes` will be converted to bytes in the subsequent step of `JsonCTemplatePolicy`, but not in the step of `JSONSerializer`. So correct the variable name to avoid misleading